src/main/java/net/mcreator/workspace/TabUtils.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2024, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.workspace;

```

Comment:
![image](https://github.com/MCreator/MCreator/assets/16374228/ee3fdc5a-d5b6-490e-949a-309454eff6bb)

Reply:
Time to add an inner `net.mcreator.generator.util` package.

Reply:
Tag and I18N utility classes are in generator package because they are involved in code generation process, whereas the new class from this PR is involved in workspace data storage, so I thought it would be appropriate to keep it where it was initially.

Reply:
Tags are also involved in code generation process

> Time to add an inner net.mcreator.generator.util package.

Can be done later, the priority of this PR is to get your other PR into 100% functional state

Reply:
> Tags are also involved in code generation process

This is what I said.

Reply:
So should I move it anyway even if it doesn't generate anything?

Reply:
I meant to say tabs.

Tabs are also something that affects generation. It generates code for tabs

Reply:
Fair point. I still see workspace root package a bit extreme, so maybe in the misc subpackage?

Reply:
Good idea.

Reply:
:heavy_check_mark: Fixed